ml/rl/workflow/identify_types_flow.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python3
+
+from typing import Dict
+
+from ml.rl.preprocessing import normalization
+from ml.rl.workflow.types import PreprocessingOptions, TableSpec
+
+import pyspark
+from pyspark.sql.functions import col, collect_list, flatten
+
+
+def get_spark_session(warehouse_dir="spark-warehouse"):
+    warehouse_dir = abspath(warehouse_dir)
```

Comment:
I think we could just leave this out. We could assume that if the user run spark locally, the warehouse location is at the directory they run the workflow from.